src/routes/safe/components/AddressBook/CreateEditEntryModal/index.tsx

```diff
@@ -42,8 +41,7 @@ const CreateEditEntryModalComponent = ({
     }
   }
 
-  const addressBook = useSelector(addressBookSelector)
-  const addressBookAddressesList = getAddressesListFromAddressBook(addressBook)
+  const addressBookAddressesList = useSelector(addressBookAddressesListSelector)
```

Comment:
nitpick: This is called `addressBookAddressesListSelector` (includes data structure name - List), but for the safe owners it's called `safeOwnersAddressesSelector`

Would be great to keep some consistency

Reply:
fixed